packages/itwinui-react/src/core/Tabs/Tabs.tsx

```diff
@@ -146,6 +184,13 @@ export const Tabs = (props: TabsProps) => {
     props = { ...props };
     delete props.actions;
   }
+  // Separate overflowOptions from props to avoid adding it to the DOM (using {...rest})
+  let overflowOptions: OverflowOptions | undefined;
+  if (props.type !== 'pill' && props.overflowOptions) {
```

Comment:
Maybe we should not allow scrolling on borderless tabs as it covers border which indicates selected tab.